<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1201</width>
    <height>1007</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>640</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>color</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>391</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="2,5,5">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>RED</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>GREEN</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>BLUE</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSpinBox" name="spinBox">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spinBox_2">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="spinBox_3">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QSlider" name="verticalSlider">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSlider" name="verticalSlider_2">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSlider" name="verticalSlider_3">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>530</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>591</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>529</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>595</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>637</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>636</x>
     <y>523</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>633</x>
     <y>595</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
